x-pack/plugins/profiling/public/components/profiling_header_action_menu.tsx

```diff
@@ -20,32 +27,38 @@ export function ProfilingHeaderActionMenu() {
 
   return (
     <EuiHeaderLinks gutterSize="xs">
-      <EuiHeaderLink
-        color="text"
-        onClick={() => {
-          const query = qs.parse(window.location.search);
-          const storageExplorerURL = url.format({
-            pathname: '/storage-explorer',
-            query: {
-              kuery: query.kuery,
-              rangeFrom: query.rangeFrom,
-              rangeTo: query.rangeTo,
-            },
-          });
-          history.push(storageExplorerURL);
-        }}
+      <EuiToolTip
+        content={i18n.translate('xpack.profiling.header.storageExplorerLink.tooltip', {
+          defaultMessage: 'This module is not GA. Please help us by reporting any bugs.',
```

Comment:
```suggestion
          defaultMessage: 'This module is not GA. If you encounter any bugs or have feedback, we’d love to hear from you. Please share your feedback via the "Give feedback" button.'',
```

Is there a standard way for users to report bugs or give feedback? The "Give feedback" button appears to be one way, but I'm not sure. To give users actionable advice, I adapted a similar tooltip from `x-pack/plugins/infra/public/pages/metrics/hosts/index.tsx` as a possible suggestion.

Reply:
@mdbirnstiehl , WDYT?

Reply:
The suggestion for actionable advice is a good idea, and I think the suggested text is good. I'm not aware of any standards for giving feedback. I would probably say "This module is in beta" instead of "This module is not GA".